module/commands/professori.py

```diff
@@ -38,17 +38,13 @@ def generate_prof_text(names: list) -> str:
 
     professors = set()
     names_number = len(names)
-    s_names = []
+    s_names = [].append(names[0])
 
     if names_number > 1:
-        s_names.append('{0} '.format(names[0]))
-
         for name in names[1:-1]:
-            s_names.append(' {0} '.format(name))
+            s_names.append('{0} '.format(name))
```

Comment:
I mean, any logic is good, but just out of curiosity:
why no spaces on the first word, suffix space on all other words and prefix space on the last word?

Reply:
First word can be the name or the first word of the lastname. In the professors table the name is at the end of the 'name' column so if you put a space in the first word and it rappresents the name we can't get the entry. Remember that in the professors table we have the column 'name' with the professor's name at the end of the string.